src/app/tests/TestCommandInteraction.cpp

```diff
@@ -224,14 +259,14 @@ class TestExchangeDelegate : public Messaging::ExchangeDelegate
     void OnResponseTimeout(Messaging::ExchangeContext * ec) override {}
 };
 
-CommandPathParams MakeTestCommandPath(CommandId aCommandId = kTestCommandId)
+CommandPathParams MakeTestCommandPath(CommandId aCommandId = kTestCommandIdWithData)
 {
     return CommandPathParams(kTestEndpointId, 0, kTestClusterId, aCommandId, (chip::app::CommandPathFlags::kEndpointIdValid));
 }
 
 void TestCommandInteraction::GenerateInvokeRequest(nlTestSuite * apSuite, void * apContext, System::PacketBufferHandle & aPayload,
                                                    bool aNeedCommandData, bool aIsTimedRequest, EndpointId aEndpointId,
-                                                   ClusterId aClusterId, CommandId aCommandId)
+                                                   ClusterId aClusterId, Optional<CommandId> aCommandId)
```

Comment:
Would suggest using a extra `aWithData` flag, Optional<CommandId> is hard to understand

Reply:
We can do that, looks like, given the limited number of callers.